src/main/java/com/google/devtools/build/lib/bazel/bzlmod/BazelModuleResolutionFunction.java

```diff
@@ -338,6 +368,7 @@ private static ImmutableMap<ModuleKey, Module> computeFinalDepGraph(
   }
 
   private static class ModuleResolutionComputeState implements Environment.SkyKeyComputeState {
+    ImmutableList<ImmutableMap<String, Optional<Checksum>>> discoveryNestedRegistryFileHashes;
```

Comment:
this is a bit of a nit, but can we move this class nearer the top? I think most other classes with State put it near the top, and it's nicer to see what state there is without having to scroll to the bottom.